src/main/kotlin/no/nav/hjelpemidler/RoutesSaksbehandling.kt

```diff
@@ -84,6 +84,15 @@ fun Route.saksbehandling() {
             val hjelpemiddeloversikt = hjelpemiddeloversiktDao.hentHjelpemiddeloversikt(fnr)
             call.respond(hjelpemiddeloversikt)
         }
+
+        post("/harUtlåntIsokode") {
+            val req = call.receive<HarUtlåntIsokodeRequest>()
+            val fnr = req.fnr
+            val isokode = req.isokode
+            validateFnr(fnr)
```

Comment:
Hvis du ser på alle andre API-endpoints i denne appen så er de wrappet i try-catch med feilhåndtering. Ktor er litt teit på dette og har ikke automatisk håndtering av Exceptions. Den bare logger noe rart om at endpointet ikke var håndtert. Har du testet dette i dev?

Noen av appene våre har satt opp en plugin som fikser dette og bruker exception type til å avgjøre feilmeldinger som logges og status kode som returneres til callee. Jeg har ellers her bare lagt inn try-catch og håndtert det rett i endpoint-handler.

Reply:
Okei, her var jeg litt rask. Ikke jeg som har skrevet all koden her. Det er tydeligvis ikke gjort for alle endpoints. Men kan være verdt å legge inn hvis du vil kontrollere hvordan dette rapporteres (feks. ved invalid fnr). Så bør det nok gjøres en oppryddingsjobb i appen på et tidspunkt. Kanskje bruke den plugin'en eller legge inn exception håndtering i alle endpoints.

Reply:
Har lagt til en try/catch nå, uansett greit å ha!